install/Knative-with-Minishift.md

```diff
@@ -10,37 +10,64 @@ You can find [guides for other platforms here](README.md).
 
 ## Before you begin
 
-These instructions will run an OpenShift 3.10 (Kubernetes 1.10) cluster on your
+These instructions will run an OpenShift 3.11 (Kubernetes 1.11) cluster on your
 local machine using [minishift](https://docs.okd.io/latest/minishift/getting-started/index.html)
 to test-drive Knative.
 
+## Minishift setup
+
+- Download the archive for your operating system from the [Minishift v1.25.0](https://github.com/minishift/minishift/releases/tag/v1.25.0) Release page and extract its contents
+- Copy the `minishift` binary from extracted directory to your preferred location like /usr/local/bin
+- Ensure `minishift` is in your PATH by running the command:
+
+```shell
+# returns minishift v1.25.0+90fb23e
+minishift version
+```
+
 ## Configure and start minishift
 
 The following details the bare minimum configuration required to setup minishift for running Knative:
 
 ```shell
+
+# make sure you have  a profile is set correctly, e.g. knative
 minishift profile set knative
+
+# Pinning to the right needed OpenShift version, anything >= v3.11.0
+minishift config set openshift-version v3.11.0
+
+# minimum memory required for the minishift VM
 minishift config set memory 8GB
+
+# the minimum required vCpus for the minishift vm
 minishift config set cpus 4
+
+# extra disk size for the vm, default is 20g
+minishift config set disk-size 50g
+
+# caching the images that will be downloaded during app deployments
 minishift config set image-caching true
-minishift addon enable admin-user
-minishift addon enable anyuid
-```
 
-The above configuration ensures that Knative gets created in its own [minishift profile](https://docs.okd.io/latest/minishift/using/profiles.html) with 8GB of RAM and 4 vCpus. The image-caching helps in re-starting up the cluster faster every time.  The [addon](https://docs.okd.io/latest/minishift/using/addons.html) **admin-user** creates a default `admin` user with the role  cluster-admin and the  [addon](https://docs.okd.io/latest/minishift/using/addons.html) **anyuid** allows the `default` service account to run the application with uid `0`.
+# Add new user called admin with password admin having role cluster-admin
```

Comment:
Current form might be misleading, adding `during Minishift start` or similar could make it more clear that user is not added in this moment, but later.

Reply:
Ah, so this is explained a little bit on line:64, so does not need a change probably.